src/sentry/web/frontend/accounts.py

```diff
@@ -110,6 +110,56 @@ def recover(request):
     return render_to_response(get_template("recover", "index"), context, request)
 
 
+@set_referrer_policy("strict-origin-when-cross-origin")
+@control_silo_function
+def relocate_reclaim(request, user_id):
+    """
+    Ask to receive a new "claim this user" email.
+    """
+
+    extra = {
+        "ip_address": request.META["REMOTE_ADDR"],
+        "user_agent": request.META.get("HTTP_USER_AGENT"),
+        "user_id": user_id,
+    }
+    if request.method != "POST":
+        logger.warning("reclaim.error", extra=extra)
+        return render_to_response(get_template("relocate", "error"), {}, request)
+
+    # Verify that the user is unclaimed. If they are already claimed, tell the requester that this
+    # is the case, since of course claiming this account would be impossible.
+    user = User.objects.filter(id=user_id).first()
```

Comment:
What prevents someone from enumerating the id space of users? Should this view have rate limiting applied to mitigate that problem?

Reply:
Added IP throttling.